.travis.yml

```diff
@@ -12,6 +12,10 @@ addons:
 before_install:
     - export DISPLAY=:99.0
     - sh -e /etc/init.d/xvfb start
+install:
+    - npm install
+    - npm install travis-github-lint-status eslint-config-standard eslint-plugin-standard eslint-plugin-promise
```

Comment:
I guess you don't need these here since they are specified in the package.json file.